sql-server-2014-alwayson-dsc/azuredeploy.parameters.json

```diff
@@ -5,6 +5,15 @@
     "newStorageAccountNamePrefix": {
       "value": ""
     },
+
```

Comment:
Please fill out values for all parameters. For things that require a unique string such as storage accounts and dns names, please fill in "#####" (5 #) to enable automated testing